engine/src/main/java/org/terasology/logic/delay/DelayedActionSystem.java

```diff
@@ -46,6 +46,12 @@
     private SortedSetMultimap<Long, EntityRef> delayedOperationsSortedByTime = TreeMultimap.create(Ordering.natural(), Ordering.arbitrary());
     private SortedSetMultimap<Long, EntityRef> periodicOperationsSortedByTime = TreeMultimap.create(Ordering.natural(), Ordering.arbitrary());
 
+    // ONLY use this for testing. DO NOT use this during regular usage.
+    public void setTime(Time t)
```

Comment:
Might this be a valid spot to use package private level access instead of public? Since the test is in the same package just under engine-tests? Can't entirely recall if it works that way.